### ISSUE TITLE
Brushes redux crash while working. 

### ISSUE THREAD

**Issue by vansciver** [tougher/Brushes#28]
Continues to crash while working on previously saved work or new work. Does not save any of the newly painted parts.

**Comment by pepysjan** [comment]
Brushes continues to crash every few minutes, when tapping, and using stylus to absorb screen colour, any tips, it's very stressful, and I'm about to do a teaching session this week!!!

**Comment by pepysjan** [comment]
If I delete and reinstall, as someone has suggested elsewhere on web, will I lose all of my work? 
Has anyone done this ?

**Comment by nfgmannaomi** [comment]
Since I did the update on brushes redux a few days ago it keeps crashing, while I'm working.  Sometimes it saves what im working on. Sometimes not up to date.  Very frustrating. Every few minutes.  I'll have to find a new drawing program.  Any suggestions? Hope they fix this.

**Comment by elianaaw01** [comment]
No, it won't save your work if you delete it. What I did was email each painting to myself to and it seemed to work. However, deleting and reinstalling it did not help the crashing issue. I've tried everything I can think of including a hard and soft reset, updating the software on my iPad, and deleting the app. Nothing worked. It is clearly a matter of the app itself not functioning right.

**Comment by nfgmannaomi** [comment]
Disappointing.  Mine was working fine til I did the update recently.  What program have you moved to? I do quick sketches with my finger on an iPad on my commute.

**Comment by londonw11** [comment]
Brushes Redux is the reduxed (reduced) version of Brushes 3. Much reduced. Practically moribund. Don't bother.  Please app writers out there, bring back the Brushes we loved and make it even better for a professional app version. I'd happily pay considerable amount for a supported app of this kind. I loved Brushes.

**Comment by marihilda** [comment]
Has anyone found a solution to this problem? The app is becoming unusable because of constantly crashing.
